File: codingjs/scoresClump.js
function scoresClump(scores: number[]): boolean {
  if (scores.length <= 2) return false;
  return scoresClumpHelper(scores, 2);
}
function scoresClumpHelper(scores: number[], index: number): boolean {
  if (index >= scores.length) return false;
  if (scores[index] - scores[index - 2] <= 2 && scores[index] - scores[index - 1] <= 2) return true
  return scoresClumpHelper(scores, ++index);
}

File: codingbat/scoresClump.java
public boolean scoresClump(int[] scores) {
  if (scores.length < 3) return false;
  for (int i = 2; i < scores.length; i++) {
    if (scores[i] - scores[i - 1] <= 2 && scores[i] - scores[i - 2] <= 2) return true;
  }
  return false;
}

